udtraek.py

```diff
@@ -100,6 +100,29 @@ def settlement(self, start_date='2000-01-01', end_date='2019-01-01'):
         return pd.DataFrame(results)
 
 
+    def multi_wdex(self, ncpus=None):
+        if ncpus is None:
+            ncpus = cpu_count() -1
+        cols = ['NYMFID', 'TRANSACTION_DATE', 'PARENT_ID', 'AMOUNT', 'FT_TYPE_FLG']
+        temp = self.df.loc[:, cols]
+        nymfids = temp['NYMFID'].unique()
+        temp['AMOUNT'] = to_amount(temp['AMOUNT'])
+        pool = ProcessingPool(ncpus=ncpus)
+        result = pool.map(lambda x: _multi_wdex_worker(temp, x), nymfids)
+        return list(filter(None,result))
```

Comment:
```suggestion
        return list(filter(None, result))
```
Læs gerne [PEP 8](https://www.python.org/dev/peps/pep-0008/).